pocs/core.py

```diff
@@ -5,6 +5,7 @@
 import warnings
 import multiprocessing
 import zmq
+import requests
```

Comment:
Not needed, was part of the slack stuff. Will pull through with a merge from upstream/develop anyway.

Reply:
OK, removing for now.